app/views/_navbar.erb

```diff
@@ -5,28 +5,48 @@
     </section>
 
     <section>
+
+      <% unless current_path == '/spaces' %>
+      <form action='/spaces' method="get">
+          <button type="submit">Available spaces</button>
+      </form>
+      <%end%>
+
+      <% if current_path == '/spaces' || current_path == '/requests' %>
```

Comment:
Let's remove this button from the navbar, move it to ```/spaces```, and rename it **List a space*** as the spec asks.